packages/react/src/Banner/Banner.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import type {Meta, StoryObj} from '@storybook/react'
+import React from 'react'
+import Link from '../Link'
+import {Banner} from '../Banner'
+import {action} from '@storybook/addon-actions'
+
+const meta = {
+  title: 'Drafts/Components/Banner',
+  component: Banner,
+} satisfies Meta<typeof Banner>
+
+export default meta
+
+export const Default = () => {
+  return (
+    <Banner
+      onDismiss={action('onDismiss')}
+      title="Info"
+      description={
+        <>
+          GitHub users are{' '}
+          <Link inline underline href="#">
+            now required
+          </Link>{' '}
+          to enable two-factor authentication as an additional security measure.
+        </>
+      }
+      primaryAction={<Banner.PrimaryAction>Button</Banner.PrimaryAction>}
+      secondaryAction={<Banner.SecondaryAction>Button</Banner.SecondaryAction>}
+    />
+  )
+}
+
+export const Playground: StoryObj<typeof Banner> = {
```

Comment:
What do you think about adding some more control options to the Playground? I think it would be nice to have...

- description (text)
- title (text)
- maaaybe do a custom icon slot, but its annoying and hard in Storybook.. 
- maybe toggle the secondary actions to show/hide?

Reply:
Great callout,  added the following:

- [x] title
- [x] description
- [x] primaryAction
- [x] secondaryAction

No icon support for now 😓 hope that's okay lol